cookbooks/bcpc/recipes/networking.rb

```diff
@@ -118,110 +118,93 @@
 bash "setup-interfaces-source" do
   user "root"
   code <<-EOH
-    echo "source /etc/network/interfaces.d/iface-*" >> /etc/network/interfaces
+    echo "source /etc/network/interfaces.d/*.cfg" >> /etc/network/interfaces
   EOH
-  not_if "grep '^source /etc/network/interfaces.d/' /etc/network/interfaces"
+  not_if "grep '^source /etc/network/interfaces.d/\*.cfg' /etc/network/interfaces"
```

Comment:
In my OpenStack cluster, I ran cluster-assign-roles 3 times and I see 3 entries for `source /etc/network/interfaces.d/\*.cfg` in `/etc/network/interfaces`. However, when I execute the not_if code on the shell, the grep does work. Not sure whats wrong.